src/reducer/admin/index.js

```diff
@@ -1,12 +1,13 @@
 import { combineReducers } from 'redux';
-import resourceReducer from './resource';
+import { createSelector } from 'reselect';
+import resourceReducer, * as resourceSelectors from './resource';
```

Comment:
I may have missed them but I failed to find collisions between the selectors here.

I suggest to explicitly import the required selectors and rename them locally only when needed. Example of renaming (probably not needed here):

```js
import resourceReducer, {
    getData,
    getRecord as getAdminResourceRecord,
    getRecordsById,
}  from './resource';
```